pytest-selenium/tests/test_toc.py

```diff
@@ -81,9 +81,12 @@ def test_toc_disables_interacting_with_content_on_mobile(selenium, base_url, boo
     toolbar.click_toc_toggle_button()
 
     # THEN: The links in the content should be disabled
-    content.assert_element_not_interactable(content.next_link)
-    content.assert_element_not_interactable(content.previous_link)
-    content.assert_element_not_interactable(attribution.attribution_link)
+    if content.next_link_is_displayed:
+        assert content.element_is_not_interactable(content.next_link)
+    if content.previous_link_is_displayed:
```

Comment:
@TomWoodward I noticed this failure in testrail and was planning to do this assert once you approve this PR.
Now thanks for updating the failing test + approving the PR :-)